src/main/kotlin/no/nav/helse/felles/Felles.kt

```diff
@@ -75,11 +71,8 @@ data class Søker(
 }
 
 data class Barn(
-    val fødselsnummer: String?,
-    val navn : String?,
-    @JsonFormat(pattern = "yyyy-MM-dd")
-    val fødselsdato: LocalDate?,
```

Comment:
Sikker på at fødselsdato skal bort?

Reply:
Det blir ikke brukt til annet enn for visning på pdf dersom fødselsnummer mangler.
Dette var tilfellet før, men nå har vi jo alltid fødselsnummer, og utleder fødselsdato fra det.

Reply:
Aha skjønner. Det vil ikke fungere hvis barnet har D-nummer, men det er kanskje et veldig corner case :P

Reply:
Hmm, godt poeng.
Det tenkte jeg ikke på. D-nummer vil ikke fungere.

Reply:
Denne snutten vil vel kræsje hvis det er tilfellet. 
https://github.com/navikt/pleiepengesoknad-prosessering/pull/100/files#diff-35be07f5e6deac004f4c816a763803ef8b4c4994e41678f23c609b1ddce8e2b4R18

Reply:
Den vil også bli litt rar dersom barnet er over 18 år, siden den tar for gitt at det er i 20** år

Reply:
Da er det fikset. 
Se 2 siste commits.